src/coreclr/debug/daccess/request.cpp

```diff
@@ -1733,7 +1733,7 @@ ClrDataAccess::GetModuleData(CLRDATA_ADDRESS addr, struct DacpModuleData *Module
     ModuleData->metadataSize = (SIZE_T) metadataSize;
 
     ModuleData->bIsReflection = pModule->IsReflection();
-    ModuleData->bIsPEFile = pModule->IsPEFile();
+    ModuleData->bIsPEFile = !pModule->GetPEAssembly()->IsDynamic();
```

Comment:
would it make sense to add a module kind enum to `Module` and replace both `Module::IsPEFile()` and `Module::IsReflection()`?

Alternately, can this be `!pModule->IsReflection()`?

Reply:
`IsReflectionEmit` would be probably the best self-describing name for `IsReflection`, `IsDynamic`, `!IsPEFile` - all these mean the same thing.